backend/controller/console.go

```diff
@@ -52,9 +59,23 @@ func (c *ConsoleService) GetModules(ctx context.Context, req *connect.Request[pb
 			case *schema.Verb:
 				//nolint:forcetypeassert
 				v := decl.ToProto().(*pschema.Verb)
+				verbSchema := schema.VerbToSchema(v)
+				dataRef := schema.DataRef{
+					Module: deployment.Module,
+					Name:   verbSchema.Request.Name,
+				}
+				jsonRequestSchema, err := schema.DataToJSONSchema(sch, dataRef)
+				if err != nil {
+					return nil, errors.WithStack(err)
+				}
+				jsonData, err := json.MarshalIndent(jsonRequestSchema, "", "  ")
+				if err != nil {
+					return nil, errors.WithStack(err)
+				}
```

Comment:
@alecthomas Does this seem right? I thought I might have some extra calls here, but maybe this is ok.